target/linux/ath79/dts/qca9531_glinet_gl-ar300m-nand.dts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "qca9531_glinet_gl-ar300m.dtsi"
+
+/ {
+	compatible = "glinet,gl-ar300m-nand", "qca,qca9531";
+	model = "GL.iNet GL-AR300M (NAND)";
+};
+
+&nand_kernel {
+	label = "kernel";
+};
```

Comment:
Similar problem here. This approach is shorter but a bit more confusing. It's more clear to put the entire mtd node in each dts instead.

Reply:
See comment above